app/main.py

```diff
@@ -1,8 +1,50 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+        self,
+        comfort_class: int,
+        clean_mark: int,
+        brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+        self,
+        distance_from_city_center: int,
+        clean_power: int,
+        average_rating: float,
+        count_of_ratings: int,
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list[Car]) -> float:
+        income = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                income += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return round(income, 1)
+
+    def calculate_washing_price(self, car: Car) -> float:
+        wash_clean_mark = self.clean_power - car.clean_mark
+        wash_price_for_comfort = car.comfort_class * wash_clean_mark
+        wash_price_for_rating = wash_price_for_comfort * self.average_rating
+        wash_price = wash_price_for_rating / self.distance_from_city_center
+        return round(wash_price, 1)
+
+    def wash_single_car(self, car: Car) -> None:
+        if self.clean_power > car.clean_mark:
+            car.clean_mark = self.clean_power
+
+    def rate_service(self, rate: int) -> None:
+        new_total_rate = (self.average_rating * self.count_of_ratings + rate)
+        new_count_of_rate = (self.count_of_ratings + 1)
```

Comment:
you can avoid creating this variable. just increase the self.count_of_ratings += 1, and than use self.count_of_ratings instead of new_count_of_rate in your calculations